labkey/utils.py

```diff
@@ -49,6 +49,11 @@ def __init__(self, **kwargs):
         else:
             self._scheme = 'http://'
 
+    def __repr__(self):
+        return '<ServerContext [ {} | {} | {} ]>'.format(self._domain,
```

Comment:
Thanks, this is great. I'd recommend switching the argument listing order of `self._container_path` and `self._context_path` so it more closely resembles what the requesting URL looks like for this context.

```py
return `<ServerContext [ {} | {} | {} ]>`.format(self._domain, self._context_path, self._container_path)
```